diff --git a/build/transform-worker/metro-transform-worker.js b/build/transform-worker/metro-transform-worker.js
index a8b84bd431cc0b311d2429bcb8f73093fa1600b4..68b8ecc2a095401f118808d011aae38e0188f444 100644
--- a/build/transform-worker/metro-transform-worker.js
+++ b/build/transform-worker/metro-transform-worker.js
@@ -446,7 +446,11 @@ async function transform(config, projectRoot, filename, data, options) {
 exports.transform = transform;
 function getCacheKey(config) {
     const { babelTransformerPath, minifierPath, ...remainingConfig } = config;
-    const filesKey = (0, metro_cache_key_1.default)([
+    const metroCacheKeySource = metro_cache_key_1.default || metro_cache_key_1;
+    const metroCacheKey = typeof metroCacheKeySource === 'function'
+        ? metroCacheKeySource
+        : metroCacheKeySource.getCacheKey;
+    const filesKey = metroCacheKey([
         require.resolve(babelTransformerPath),
         require.resolve(minifierPath),
         require.resolve('metro-transform-worker/src/utils/getMinifier'),
